python/dgl/distributed/graph_partition_book.py

```diff
@@ -646,15 +682,15 @@ class RangePartitionBook(GraphPartitionBook):
         the number of partitions. Each row has two integers: the starting and the ending IDs
         for a particular node type in a partition. For example, all nodes of type ``"T"`` in
         partition ``i`` has ID range ``node_map["T"][i][0]`` to ``node_map["T"][i][1]``.
-    edge_map : dict[str, Tensor]
+    edge_map : dict[str, Tensor] or dict[(str, str, str), Tensor]
         Global edge ID ranges within partitions for each edge type. The key is the edge type
         name in string. The value is a tensor of shape :math:`(K, 2)`, where :math:`K` is
         the number of partitions. Each row has two integers: the starting and the ending IDs
         for a particular edge type in a partition. For example, all edges of type ``"T"`` in
         partition ``i`` has ID range ``edge_map["T"][i][0]`` to ``edge_map["T"][i][1]``.
     ntypes : dict[str, int]
         map ntype strings to ntype IDs.
-    etypes : dict[str, int]
+    etypes : dict[str, int] or dict[(str, str, str), int]
         map etype strings to etype IDs.
```

Comment:
Add a deprecation section below about `edge_map` and `etypes` will only support triplet keys in the future.

Reply:
added.